main.go

```diff
@@ -16,6 +18,14 @@ func main() {
 	go service.StartRunnerService(brain, errChan)
 	go service.StartTaskService(brain, errChan)
 
+	// runtime.SetMutexProfileFraction(2)
```

Comment:
Leaving this for now so I can set up a flag for it later, when I convert it to cobra